src/main/java/service/impl/FileServiceImplementation.java

```diff
@@ -0,0 +1,34 @@
+package service.impl;
+
+import au.com.bytecode.opencsv.CSVReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import service.FileService;
+
+public class FileServiceImplementation implements FileService {
+
+    public List<String[]> readFile(String filePath) {
+        File file = new File(filePath);
+        try (CSVReader reader = new CSVReader(new FileReader(file))) {

```

Comment:
Rename this implementation of FileReader interface according to the files you are working with here